### DISCUSSION TITLE
ip a and ip route commands returning errors

### DISCUSSION THREAD

**Discussion by msovara** [chpc-tech-eval/scc#38] [General]
When running the ip and ip route commands, the shell returns the error "command not found". I am running MobaXterm on a Windows machine.

**Comment by msovara** [comment]
Same error message using tracepath and tracert commands.

**Reply by ZamaZethu** [reply]
please share the screenshot

**Reply by ZamaZethu** [reply]
install them : **sudo dnf install iproute traceroute -y**

traceroute and tracert are the same command

**Reply by The5upercelln1nja** [reply]
> Same error message using tracepath and tracert commands.
Unisa-sct24
Try typing man tracepath tracepath is used in unix systems type tracepath ip address it will work

**Comment by msovara** [comment]
![MobaXterm screenshot](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/18695654/805f9928-1638-4eb8-8488-849333c9422b)

**Reply by ZamaZethu** [reply]
@msovara you need to install them as shown above. 

syntax:
**tracepath IP** 

Please use man page to see more options :  **man tracepath**

**Comment by msovara** [comment]
Some things to note about MobaXterm: It is a popular terminal emulator and SSH client for the Windows environment, but it may not include all the utilities and tools found in a full Linux distribution by default. It typically does not have the "dnf" package manager or "man" (manual) pages. Although MobaXterm offers plugins that can extend its functionality, for a more complete Unix-like experience, it may be better to use the Windows Subsystem for Linux (WSL) or Cygwin.

**Reply by ZamaZethu** [reply]
@msovara I thought you are on your headnode running Rocky, **dnf is package manager for rocky linux**

**Reply by msovara** [reply]
Thank you for the above comments Zama. I cloned the GitHub repository to my local machine and tested the commands there. I am now testing on Sebowa.
